templates/firefox-ios.xliff

```diff
@@ -2134,6 +2138,10 @@
         <source>When Opening Firefox</source>
         <note>Description displayed under the ”Offer to Open Copied Link” option. See https://bug1223660.bmoattachments.org/attachment.cgi?id=8898349</note>
       </trans-unit>
+      <trans-unit id="Settings.OfferClipboardBar.StatusV2" xml:space="preserve">
+        <source>When opening Firefox</source>
```

Comment:
No, it is not an experiment. The only thing that I did here, is to change one word, from "Opening" to "opening".

Reply:
And also, this setting already existed before. I only replaced that word.

Reply:
That's probably because the old string landed before we put the linter in place. Right now as you can see in this PR, the linter is catching "Firefox" as an [error](https://github.com/mozilla-l10n/firefoxios-l10n/actions/runs/9421580674/job/25955962746?pr=228).
Best practice is to hardcoded brand names, like "Firefox" (but if it's a feature name containing "Firefox" it's ok to leave it hardcoded).
Please replace the app name with a placeholder (in the meantime, I'll check where this is documented)

Reply:
@Delphine this has been addressed. Please let me know if you have any other issues.
Thank you!

Reply:
@tusharC95 

Not sure why we ended up with 2 strings here: `Settings.OfferClipboardBar.Status.v128` and `Settings.OfferClipboardBar.Status.v2`.

This is the pull request that introduced the v128: https://github.com/mozilla-mobile/firefox-ios/pull/20606

I suppose the problem is that the v2 string is used in fx127?

I'll see if I can work around it. Note that the string [could have been changed without a new ID](https://github.com/mozilla-l10n/nimbus-l10n/issues/34), since it was only changing capitalization: https://mozilla-l10n.github.io/documentation/localization/making_string_changes.html